doc/man3/SSL_CTX_set_options.pod

```diff
@@ -214,6 +214,15 @@ functionality is not required. Those applications can turn this feature off by
 setting this option. This is a server-side opton only. It is ignored by
 clients.
 
+=item SSL_OP_NO_CERTIFICATE_COMPRESSION
+
+Normally clients and servers will transparently attempt to negotiate the
+RFC8879 certificate compression option on TLSv1.3 connections.
+
+If this option is set, certificate compression is disabled. The extension
+will not be sent by either client or server, and will be ignored upon
+receipt.
```

Comment:
Each endpoint can control (via the callback) whether to actually compress their own certificate.
In terms of the extension, it's really an indication of support; there's no way to say I can send compressed certificates without also receiving them. There's a single setting, not a per-client/per-server setting.

Reply:
To clarify from the RFC:

> By sending a compress_certificate extension, the sender indicates to the peer the certificate-compression algorithms it is willing to use for decompression.

> The compress_certificate extension is a unidirectional indication; no corresponding response extension is needed.

So, this indicates our willingness to decompress, we compress if the peer indicates it can decompress, we have this extension enabled, and the callback does not bypass compression.